diff --git a/arch/arm64/boot/dts/rockchip/rk3588-turing-rk1.dtsi b/arch/arm64/boot/dts/rockchip/rk3588-turing-rk1.dtsi
index dbaa94ca69f4..1e4d2d3c469f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-turing-rk1.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3588-turing-rk1.dtsi
@@ -65,6 +65,47 @@ vcc_1v1_nldo_s3: vcc-1v1-nldo-s3-regulator {
        };
 };
 
+&package_thermal {
+       polling-delay = <1000>;
+
+       trips {
+               trip_passive: trip_passive {
+                       temperature = <40000>;
+                       hysteresis = <2000>;
+                       type = "passive";
+               };
+    trip_active: trip_active {
+                       temperature = <45000>;
+                       hysteresis = <2000>;
+                       type = "active";
+               };
+    trip_hot: trip_hot {
+                       temperature = <80000>;
+                       hysteresis = <2000>;
+                       type = "hot";
+               };
+       };
+
+       cooling-maps {
+               map0 {
+                       trip = <&trip_passive>;
+                       cooling-device = <&fan 0 1>;
+               };
+    map1 {
+      trip = <&trip_active>;
+      cooling-device = <&fan 1 4>;
+    };
+    map2 {
+      trip = <&trip_hot>;
+      cooling-device = <&fan 4 5>;
+    };
+    map3 {
+      trip = <&package_crit>;
+      cooling-device = <&fan 5 5>;
+    };
+       };
+};
+
 &combphy2_psu {
        status = "okay";
 };

